slask.py

```diff
@@ -17,15 +17,15 @@
 import debt_utils
 from config import config
 
-def handle_message(message):
+def handle_message(command):
     # 'help'
```

Comment:
I mean, it isn't a _command_ until we've parsed it and figured out if it's telling us anything? Also if I was going to rename the param I'd also rename the function. 😛